### DISCUSSION TITLE
Zapi XML Issue

### DISCUSSION THREAD

**Discussion by swutter** [NetApp/harvest#833] [Q&A] [answered]
Hey All,

I was using a fairly old version of Harvest and upgraded with some heavy lifting but am running into some issues with the Zapi and ZapiPerf XML files.  When I start harvest I get the following two errors:

init collector-object (Zapi:Node): connection error => XML syntax error on line 1: attribute name without = in element
init collector-object (ZapiPerf:SystemNode): connection error => XML syntax error on line 1: attribute name without = in element

The two yaml files being referenced are the default files from the repo so I'm not sure why an error for line 1 is being thrown.  I feel like I'm missing something simple.

Thanks in advance!

**Comment by cgrinds** [comment]
hi @swutter what version of Harvest are you running? Any errors in your poller log files (typically `/var/log/harvest`)

I wonder if those XML errors are from Harvest talking to something that isn't ONTAP? ie. Harvest is trying to talk ZAPI to something that doesn't talk ZAPI? Can you run `bin/harvest doctor` and `bin/harvest doctor --print` and see if any errors are reported?

Can you try running a single poller, monitoring a single cluster? There's more troubleshooting information [here](https://github.com/NetApp/harvest/wiki/Troubleshooting-Harvest).

**Reply by swutter** [reply]
Hey @cgrinds , I'm running the newest version.  I also realized from your comment that my poller should be pointed to a cluster and not an OCUM. I changed my config and it started running.  I'm getting permission and poller errors now which I'll look more into.  

Should I be including OCUM (ActiveIQ) and the Cluster info now?  If so I don't see a way to tell the difference in the config file.

**Comment by cgrinds** [comment, marked as answer]
Hey @swutter thanks for the follow up, that makes sense. OCUM is no longer used by Harvest - no need to include it. In terms of permissions and auth that's documented [here](https://github.com/NetApp/harvest/blob/main/docs/AuthAndPermissions.md#authentication).
